main.go

```diff
@@ -68,6 +59,22 @@ func main() {
 	}
 }
 
+// Install custom signal handlers:
+func installSignalHandlers(ctx context.Context, m *firecracker.Machine) {
```

Comment:
nit: I'd put this below `runVMM`; I tend to like caller-above-callee (it matches the way I like to read code; the most abstract thing first and then drill down into the most concrete thing later).

Reply:
Yep, I agree with that style. Updated.